### PR TITLE
Add istat-menus version 5

### PR DESCRIPTION
**Pull request by robdecker**
Recently Bjango released iStat Menus 6. This brings in iStat Menus 5 for those of us who don't want to upgrade.

This comes from https://github.com/caskroom/homebrew-cask/blob/3a68e11fa578e36fe4745d761a6082b0b158391b/Casks/istat-menus.rb

### CODE REVIEW

File: Casks/istat-menus5.rb [hunk 1]
```diff
@@ -0,0 +1,40 @@
+cask 'istat-menus5' do
+  version '5.32'
+  sha256 :no_check # required as upstream package is updated in-place
```
Comment (commitay) [inline Casks/istat-menus5.rb]: We can try re-adding a `sha256`, hopefully `5.32` won't be updated in-place.

**Comment by robdecker** [comment]
@commitay could you confirm the sha256 is `bfb1cd95d119c4248190dc6093147a2b08a04b27b9e57e90638d720f7b2f24c7`?

**Comment by commitay** [comment]
```
bfb1cd95d119c4248190dc6093147a2b08a04b27b9e57e90638d720f7b2f24c7  istatmenus5.32.zip
```

**Comment by miccal** [comment]
Thank you for your contribution.

**Pull request merged**